src/System.Management.Automation/engine/parser/PSType.cs

```diff
@@ -1100,30 +1102,42 @@ internal void DefineEnum()
             }
         }
 
-        private static IEnumerable<CustomAttributeBuilder> GetAssemblyAttributeBuilders()
+        private static IEnumerable<CustomAttributeBuilder> GetAssemblyAttributeBuilders(string scriptFile)
         {
-            yield return new CustomAttributeBuilder(typeof(DynamicClassImplementationAssemblyAttribute).GetConstructor(Type.EmptyTypes), s_emptyArgArray);
+            var ctor = typeof(DynamicClassImplementationAssemblyAttribute).GetConstructor(Type.EmptyTypes);
+            var emptyArgs = Utils.EmptyArray<object>();
```

Comment:
You call `Utils.EmptyArray<object>()` multiple times, why not just use `s_emptyArgArray`?

Reply:
I'll remove s_emptyArgArray - it's probably not better to store 2 references to the same array, `Utils.EmptyArray` already has a static reference.